### ISSUE TITLE
Claims accessible to all

### ISSUE THREAD

**Issue by tbstewa** [GriefPrevention/GriefPrevention#83]
### What behaviour is expected:
Claims/chests to be protected
### What behaviour is observed:
Players without access are able to edit terrain and chest contents of other players. Claims don't exist when examining specific blocks but show when examining the area. 
### Steps/models to reproduce:
claim land and have an untrusted person use a chest or edit the terrain
### Spigot/Craftbukkit/Paper and GriefPrevention version:

git-paper-1051 1.11.2
GP version 16.6

### Stack trace/error (if applicable)

```
Paste entire stack trace inside here
```

### GriefPrevention config.yml (if applicable)

```yml
# Default values are perfect for most servers.  If you want to customize and have a question, look for the answer here first: http://dev.bukkit.org/bukkit-plugins/grief-prevention/pages/setup-and-configuration/
GriefPrevention:
  SeaLevelOverrides:
    world: -1
    world_nether: -1
    world_the_end: -1
  Claims:
    Mode:
      world: Survival
      world_the_end: Disabled
      world_nether: Disabled
    PreventTheft: true
    ProtectCreatures: true
    PreventButtonsSwitches: true
    LockWoodenDoors: true
    LockTrapDoors: true
    LockFenceGates: true
    EnderPearlsRequireAccessTrust: true
    ProtectHorses: true
    InitialBlocks: 256
    Claim Blocks Accrued Per Hour:
      Default: 100
    Max Accrued Claim Blocks:
      Default: 25600
    Accrued Idle Threshold: 0
    AbandonReturnRatio: 1.0
    AutomaticNewPlayerClaimsRadius: 4
    ExtendIntoGroundDistance: 5
    MinimumWidth: 5
    MinimumArea: 100
    MaximumDepth: 256
    InvestigationTool: STICK
    ModificationTool: GOLD_SPADE
    Expiration:
      ChestClaimDays: 7
      UnusedClaimDays: 14
      AllClaims:
        DaysInactive: 60
        ExceptWhenOwnerHasTotalClaimBlocks: 10000
        ExceptWhenOwnerHasBonusClaimBlocks: 5000
      AutomaticNatureRestoration:
        SurvivalWorlds: false
    AllowTrappedInAdminClaims: true
    MaximumNumberOfClaimsPerPlayer: 10
    CreationRequiresWorldGuardBuildPermission: true
    PortalGenerationRequiresPermission: true
    VillagerTradingRequiresPermission: true
    CommandsRequiringAccessTrust: /sethome
    DeliverManuals: true
    ManualDeliveryDelaySeconds: 30
  Spam:
    Enabled: false
    LoginCooldownSeconds: 60
    ChatSlashCommands: /me;/global;/local
    WhisperSlashCommands: /tell;/pm;/r;/whisper;/msg
    WarningMessage: Please reduce your noise level.  Spammers will be banned.
    BanOffenders: true
    BanMessage: Banned for spam.
    AllowedIpAddresses: 1.2.3.4; 5.6.7.8
    DeathMessageCooldownSeconds: 120
    Logout Message Delay In Seconds: 0
  PvP:
    RulesEnabledInWorld:
      world: false
      world_nether: false
      world_the_end: false
    ProtectFreshSpawns: true
    PunishLogout: true
    CombatTimeoutSeconds: 15
    AllowCombatItemDrop: false
    BlockedSlashCommands: /home;/vanish;/spawn;/tpa
    ProtectPlayersInLandClaims:
      PlayerOwnedClaims: false
      AdministrativeClaims: true
      AdministrativeSubdivisions: true
    AllowLavaDumpingNearOtherPlayers: true
    AllowFlintAndSteelNearOtherPlayers: true
    ProtectPetsOutsideLandClaims: false
  Economy:
    ClaimBlocksPurchaseCost: 0.0
    ClaimBlocksSellValue: 0.0
  ProtectItemsDroppedOnDeath:
    PvPWorlds: false
    NonPvPWorlds: false
  BlockLandClaimExplosions: true
  BlockSurfaceCreeperExplosions: true
  BlockSurfaceOtherExplosions: true
  LimitSkyTrees: true
  LimitTreeGrowth: false
  LimitPistonsToLandClaims: true
  FireSpreads: false
  FireDestroys: false
  AdminsGetWhispers: false
  AdminsGetSignNotifications: false
  SmartBan: false
  Mute New Players Using Banned Words: false
  MaxPlayersPerIpAddress: 3
  Siege:
    Worlds: []
    BreakableBlocks:
    - DIRT
    - GRASS
    - LONG_GRASS
    - COBBLESTONE
    - GRAVEL
    - SAND
    - GLASS
    - THIN_GLASS
    - WOOD
    - WOOL
    - SNOW
  EndermenMoveBlocks: false
  SilverfishBreakBlocks: false
  CreaturesTrampleCrops: false
  RabbitsEatCrops: false
  HardModeZombiesBreakDoors: false
  Database:
    URL: ''
    UserName: ''
    Password: ''
  UseBanCommand: false
  BanCommandPattern: ban %name% %reason%
  Mods:
    BlockIdsRequiringAccessTrust: []
    BlockIdsRequiringContainerTrust:
    - 99999:*:Example - ID 99999, all data values.
    BlockIdsExplodable: []
    PlayersIgnoringAllClaims: []
  Abridged Logs:
    Days To Keep: 7
    Included Entry Types:
      Social Activity: true
      Suspicious Activity: true
      Administrative Activity: false
      Debug: false
      Muted Chat Messages: false

```

**Comment by RoboMWM** [comment]
>  Claims don't exist when examining specific blocks but show when examining the area.

Can you elaborate? Also, try without any other plugins installed.

**Comment by tbstewa** [comment]
I don't think there was anything to elaborate on. a player claimed an area and when inspecting the block it said nobody had claimed it. When I did a shift+right click with the stick, it showed the area's borders and said it found 1 claim. The way I fixed it was downloading my plugins folder, making a local server, starting it and then deleting all the claims in the world. Then I tested some claims and transfers, then reuploaded it. The plugin is now working as expected.
